test/Model.js

```diff
@@ -48,12 +54,12 @@ describe("Model", () => {
 
     const parsedAuditOutput = {
       advisories: {
-        1066786: {
-          id: 1066786,
+        1004291: {
+          id: 1004291,
```

Comment:
I updated these while you made these changes. Even though it's deprecated, we might as well keep it as correct as possible. Please revert these ID changes 😄